corehq/motech/openmrs/serializers.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import datetime
+import re
+
+import six
+from dateutil import parser as dateutil_parser
+
+
+def to_timestamp(value):
+    """
+    OpenMRS accepts strings in the same format for both dates and datetimes.
+
+    >>> to_timestamp('2017-06-27')
+    '2017-06-27T00:00:00.000+0000'
+
+    """
+    if isinstance(value, six.string_types):
+        value = dateutil_parser.parse(value)
```

Comment:
just curious if you think this should be more strict in the types of values it allows to parse. i've seen parse() be very lenient - for example, if you pass it '1' it will just give you the first day of the current month with the current year

Reply:
Interesting. I didn't know dateutil's parser was so lenient.

I think it's reasonable to expect app builders to send dates and datetimes that start with 'YYYY-MM-DD'.